styles/linter-plus.less

```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```

Comment:
Aha got it, I took that out of context. Why is display: block needed on the div elements created here?